kontrakt/src/main/java/no/nav/abakus/iaygrunnlag/inntektsmelding/v1/RefusjonskravDatoerDto.java

```diff
@@ -21,16 +21,25 @@
     @Valid
     private List<RefusjonskravDatoDto> refusjonskravDatoer;
 
+    @JsonProperty(value = "refusjonskravDatoerPrArbeidsforhold", required = true)
+    @NotNull
+    @Valid
+    private List<RefusjonskravDatoDto> refusjonskravDatoerPrArbeidsforhold;
```

Comment:
Er dette innkommende data? Isåfall, hvordan forhindrer du at den er null før du har satt feltet hos avsender? Alle som tar i bruk ny kontrakt nå og ikke setter dette feltet vil vel knekke koden?